tutorials/katacoda/basic-pipeline/index.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Tekton Basic Pipeline",
+  "description": "Learn how to build a basic Tekton Pipeline!",
+  "difficulty": "Beginner",
+  "time": "30",
+  "details": {
+    "steps": [
+      {
+        "title": "Create a basic Task",
+        "text": "step1.md",
+        "code": "init.sh"
+      },
+      {
+        "title": "Create a second basic Task",
+        "text": "step2.md"
+      },
+      {
+        "title": "Create a Pipeline and PipelineRun",
+        "text": "step3.md"
+      }
+    ],
+    "intro": {
+      "text": "intro.md"
+    },
+    "finish": {
+      "text": "finish.md"
+    },
+    "assets": {
+      "host01": [
+        {
+          "file": "*",
+          "target": "~/"
+        }
+      ]
+    }
+  },
+  "environment": {
+    "uilayout": "terminal-iframe"
+  },
+  "backend": {
+    "imageid": "kubernetes",
```

Comment:
`kubectl apply -f https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.13.2/release.yaml` is failing with

```
error: error validating "https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.13.2/release.yaml": error validating data: [ValidationError(CustomResourceDefinition.spec): unknown field "preserveUnknownFields" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.CustomResourceDefinitionSpec, ValidationError(CustomResourceDefinition.spec.validation.openAPIV3Schema): unknown field "x-kubernetes-preserve-unknown-fields" in io.k8s.apiextensions-apiserver.pkg.apis.apiextensions.v1beta1.JSONSchemaProps]; if you choose to ignore these errors, turn validation off with --validate=false
```

```
master $ kubectl version
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:53:57Z", GoVersion:"go1.12.1",Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.0", GitCommit:"641856db18352033a0d96dbc99153fa3b27298e5", GitTreeState:"clean", BuildDate:"2019-03-25T15:45:25Z", GoVersion:"go1.12.1",Compiler:"gc", Platform:"linux/amd64"}
```

Since Tekton Pipelines 0.11 the minimum required kubernetes version is 1.15.
You can update the katacoda scenario to use kubernetes 1.18 by specifying:
`"imageid": "kubernetes-cluster-running:1.18"`

Reply:
Fixed :)